weld_rt/cpp/Makefile

```diff
@@ -1,7 +1,7 @@
 OS = $(shell uname -s)
 LLVM_VERSION = $(shell llvm-config --version | cut -d . -f 1,2)
 
-CFLAGS = -O3 -std=c++11 -Wall -fno-use-cxa-atexit -fPIC -flto
+CFLAGS = -O0 -g -std=c++11 -Wall -fno-use-cxa-atexit -fPIC -flto
```

Comment:
Yeah this is required for now, hopefully when @mateiz converts this to a shared library the problem goes away and we can revert to -O3. It shouldn't be a big problem even now because MCJIT further optimizes this code